src/Kunstmaan/MultiDomainBundle/Tests/Helper/DomainConfigurationTest.php

```diff
@@ -318,6 +317,24 @@ private function getEntityManager()
         return $em;
     }
 
+    private function getAdminRouteHelper()
```

Comment:
@Numkil this test is useless as you are supposed to test `AdminRouteHelper`, but not mock it :(